src/patternfly/components/MenuToggle/menu-toggle.scss

```
@@ -337,6 +341,33 @@
   &.pf-m-danger {
     --#{$menu-toggle}--BorderColor: var(--#{$menu-toggle}--m-danger--BorderColor);
     --#{$menu-toggle}__status-icon--Color: var(--#{$menu-toggle}--m-danger__status-icon--Color);
+
+    @media screen and (prefers-reduced-motion: no-preference) {
+      transform: translate3d(0, 0, 0);
```

Just a note that you can change this to `translateX()`. You could also go ahead and put a var here, then just modify the var in the animation `@keyframes` if you wanted.

```suggestion
      transform: translate3d(0, 0, 0);
```